XRTK-Core/Packages/com.xrtk.core/Definitions/Platforms/IOSPlatform.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) XRTK. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace XRTK.Definitions.Platforms
+{
+    /// <summary>
+    ///  Used by the XRTK to signal that the feature is available on the iOS platform.
+    /// </summary>
+    public class IOSPlatform : BasePlatform
```

Comment:
See comment above. It's kind of random that most platforms are defined in Core and some not.

Reply:
We might move these out as we get platform specific sdks.  I thought it was fine for now since we don't have an iOS submodule